README.md

```diff
@@ -169,9 +169,24 @@ make migrate-init
    
 ## Future Node updates
 
-Upon any upcoming Insolar MainNet updates, the current Node version may suspend synching. To resume the synching, you need to update your Node instance.
+Upon any upcoming Insolar MainNet updates, the current Node version may suspend synching. To resume synching, you need to update your Node instance.
 
-Update instructions for future versions of the Node are on the way and will be released closer to the next significant update. 
+Each time your Node instance connects to the trusted Heavy Material Node, your Node instance sends its client type (`client_type`), communication protocol version (`heavy_version`) and smart contract version (`contract_version`) in the metadata.
```

Comment:
Metadata of what? Just drop it. Also, Oxford comma.
Suggestion:
Each time your Node instance connects to the trusted Heavy Material Node (HMN), the instance sends its client type (`client_type`), communication protocol version (`heavy_version`), and smart contract version (`contract_version`) to the node.

Reply:
Each time your Node instance connects to the trusted Heavy Material Node, the instance sends its client type (`client_type`), communication protocol version (`heavy_version`), and smart contract version (`contract_version`) to this Heavy Material Node.

Reply:
"the node" at the end may be confusing seeing how we call the Observer itself "the Node".

Reply:
Overused Heavy Material Node throughout the instructions, didn't I? You opinion?

Reply:
A little, yeah